api/lib/domain/usecases/import-students.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async ({ organizationId, file, studentsXmlService, studentRepository }) => {
```

Comment:
Attention, une règle de formatage des _usecase_ (ou des modules qui exportent une fonction, en général) qu'on a[vait écrit quelque part un jour 🤷‍♂] c'est de faire apparaître le nom canonique de la fonction dans le fichier :

```suggestion
module.exports = async function importStudents({ organizationId, file, studentsXmlService, studentRepository }) {
```

pour pouvoir la retrouver facilement avec un `grep`.

Il y a effectivement quelques autres _usecases_ qui ne respectent pas cette règle mais ça ne devrait pas durer : https://github.com/1024pix/pix/pull/681 😉

Reply:
done !